test/addons/openssl-client-cert-engine/test.js

```diff
@@ -44,16 +44,10 @@ const server = https.createServer(serverOptions, common.mustCall((req, res) => {
   };
 
   const req = https.request(clientOptions, common.mustCall((response) => {
-    let body = '';
-    response.setEncoding('utf8');
-    response.on('data', (chunk) => {
-      body += chunk;
-    });
-
-    response.on('end', common.mustCall(() => {
+    common.collectStream(response).then((body) => {
       assert.strictEqual(body, 'hello world');
       server.close();
-    }));
+    });
```

Comment:
The `then()` handler should be wrapped in a `common.mustCall()`

Reply:
Thanks, missed that.